rel/i18n/emqx_conf_schema.hocon

```diff
@@ -480,10 +480,12 @@ desc_log_throttling.label:
 
 desc_log_throttling.desc:
 """Log throttling feature reduces the number of potentially flooding logged events by
-dropping all but the first event within a configured time window."""
+dropping all but the first event within a configured time window.
+The throttling is automatically disabled if the primary log level is set to debug."""
```

Comment:
there is no 'primary log level' concept exposed to users.

```suggestion
dropping all but the first event within a configured time window.
The throttling is automatically disabled if `console` or `file` log level is set to `debug`."""
```

Reply:
Indeed, my bad.. Fixed